aries-site/src/examples/templates/page-layouts/anatomy/PageMarginAnatomy.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Box, Grid, Stack, Text } from 'grommet';
+import PropTypes from 'prop-types';
+
+// values chosen to best visually represent the margin area in diagram
+const largeColumns = ['13px', 'auto', '13px'];
```

Comment:
wondering if we can use theme values of edgeSize to get close here.

edgeSize.medium = 12px;
edgeSize.small = 6px;

If we use these values, is it reasonably close?

Reply:
think that would be fine since their reasonably close